client/src/components/layouts/certification.tsx

```diff
@@ -1,47 +1,48 @@
-import React, { Component } from 'react';
-import PropTypes from 'prop-types';
+import React, { Fragment, Component } from 'react';
 import { connect } from 'react-redux';
 
 import { fetchUser, isSignedInSelector, executeGA } from '../../redux';
 import { createSelector } from 'reselect';
 import Helmet from 'react-helmet';
 
+interface CertificationProps {
+  children?: React.ReactNode;
+  executeGA?: (args: { type: string; data: string }) => void;
+  fetchUser: () => void;
+  isSignedIn?: boolean;
+  pathname: string;
+}
+
 const mapStateToProps = createSelector(isSignedInSelector, isSignedIn => ({
   isSignedIn
 }));
 
 const mapDispatchToProps = { fetchUser, executeGA };
 
-class CertificationLayout extends Component {
+class CertificationLayout extends Component<CertificationProps> {
+  static displayName = 'CertificationLayout';
   componentDidMount() {
     const { isSignedIn, fetchUser, pathname } = this.props;
     if (!isSignedIn) {
       fetchUser();
     }
-    this.props.executeGA({ type: 'page', data: pathname });
+    if (this.props.executeGA) {
+      this.props.executeGA({ type: 'page', data: pathname });
+    }
   }
 
-  render() {
+  render(): JSX.Element {
     const { children } = this.props;
 
     return (
-      <>
+      <Fragment>
```

Comment:
```suggestion
      <>
```
We recently changed the eslint rules to require the shorthand: https://github.com/freeCodeCamp/freeCodeCamp/pull/42963/